### PR TITLE
Implementation of min and max atributes for repeating type

### PR DESCRIPTION
**Pull request by icrc-jofrancisco**
In AMPATH module, the "repeating" type must support both minimum and maximum attributes.

- The minimum attribute represents the minimum number of input sections supported for that group. The entries of the group must open automatically with the minimum number.

- Eg: "min": 3 , will show 3 input sections when the form is opened.

- When the user tries to remove an element and the minimum limit is reached, the section of the "removed" button should be cleared and not removed.

- The max attribute should limit the number of input sections. When the maximum is reached, the "Add" button should be marked as "disabled".

- It should be added below the group title, the minimum and maximum possible values for this group (if available).

min_max_repeating

### CODE REVIEW

File: projects/ngx-formentry/src/form-entry/form-renderer/form-renderer.component.html [hunk 1]
```diff
@@ -470,4 +473,4 @@ <h4 style="margin: 2px; font-weight: bold">{{ node.question.label }}</h4>
       ></form-renderer>
     </div>
   </div>
-</div>
+</div>
```
Comment (ibacher) [inline projects/ngx-formentry/src/form-entry/form-renderer/form-renderer.component.html:476]:
```suggestion
</div>

```

File: yarn.lock [hunk 2]
```diff
@@ -11302,4 +11302,4 @@ yocto-queue@^0.1.0:
 zone.js@~0.10.2:
   version "0.10.3"
   resolved "https://registry.yarnpkg.com/zone.js/-/zone.js-0.10.3.tgz#3e5e4da03c607c9dcd92e37dd35687a14a140c16"
-  integrity sha512-LXVLVEq0NNOqK/fLJo3d0kfzd4sxwn2/h67/02pjCjfKDxgx1i9QqpvtHD8CrBnSSwMw5+dy11O7FRX5mkO7Cg==
+  integrity sha512-LXVLVEq0NNOqK/fLJo3d0kfzd4sxwn2/h67/02pjCjfKDxgx1i9QqpvtHD8CrBnSSwMw5+dy11O7FRX5mkO7Cg==
```
Comment (ibacher) [inline yarn.lock:11305]:
```suggestion
  integrity sha512-LXVLVEq0NNOqK/fLJo3d0kfzd4sxwn2/h67/02pjCjfKDxgx1i9QqpvtHD8CrBnSSwMw5+dy11O7FRX5mkO7Cg==
  
```

**Comment by gracepotma** [comment]
Discussed briefly with @ibacher today for a status update: Ian plans to review this in detail next week.

**Comment by ibacher** [comment]
I think everything looks fine except we should fix the failing test before this gets merged in.

**Comment by icrc-jofrancisco** [comment]
> I think everything looks fine except we should fix the failing test before this gets merged in.

> 

Hi @ibacher , 
The tests are now fixed! Thanks

**Review by ibacher — APPROVED** [review]

**Comment by ibacher** [comment]
Thanks @icrc-jofrancisco!

**Pull request merged**